### ISSUE TITLE
parcoords line coloring error with integer TypedArray

### ISSUE THREAD

**Issue by jonmmease** [plotly/plotly.js#3595]
There seems to be an issue with specifying line color of a `parcoords` trace with an integer TypedArray. All of the values seem to be rounded to the min or max color value.

CodePen: https://codepen.io/jonmmease/pen/QoNgMe?editors=1010
```javascript
var trace = {
  type: 'parcoords',
  line: {
	  color: new Int32Array([0, 1, 2])
	  // color: new Float32Array([0, 1, 2])
          // color: [0, 1, 2]	  
  },
	
  dimensions: [{
    range: [1, 5],
    label: 'A',
    values: [1,4,3]
  }, {    
    range: [1,5],
    label: 'B',
    values: [3,1.5,2],
  }, {
    range: [1, 5],
    label: 'C',
    values: [2,4,1],
  }]
};

var data = [trace]

Plotly.plot('graphDiv', data, {}, {showSendToCloud: true});
```

When `line.color` is specified as a standard array or floating point TypedArray:

![newplot 4](https://user-images.githubusercontent.com/15064365/53681210-37d50d00-3cb4-11e9-887e-b295a745ddde.png)

When `line.color` is specified as an integer TypedArray:
![newplot 3](https://user-images.githubusercontent.com/15064365/53681214-46bbbf80-3cb4-11e9-804c-290105f47fea.png)

**Comment by archmoj** [comment]
@jonmmease looking at `parcoords` code I thought a similar issue may be existed for `values`. And yes here is [codepen](https://codepen.io/MojtabaSamimi/pen/EMyNRB?editors=1010) demonstrating that.

**Comment by archmoj** [comment]
> @jonmmease looking at `parcoords` code I thought a similar issue may be existed for `values`. And yes here is [codepen](https://codepen.io/MojtabaSamimi/pen/EMyNRB?editors=1010) demonstrating that.

Which would be also addressed in PR #3598.

**Comment by etpinard** [comment]
fixed in https://github.com/plotly/plotly.js/pull/3598
